templates/repo/actions/status.tmpl

```diff
@@ -1,12 +1,26 @@
-{{if .IsWaiting}}
-	<i class="commit-status circle icon gray"></i>
-{{end}}
-{{if .IsRunning}}
-	<i class="commit-status circle icon yellow"></i>
-{{end}}
-{{if .IsSuccess}}
-	<i class="commit-status check icon green"></i>
-{{end}}
-{{if .IsFailure}}
-	<i class="commit-status warning icon red"></i>
+<!-- This template should be kept the same as web_src/js/components/ActionRunStatus.vue
+	Please also update the vue file above if this template is modified.
+-->
+{{- $size := 16 -}}
+{{- if .size -}}
+{{- $size = .size -}}
+{{- end -}}
+
+{{- $className := "" -}}
+{{- if .className -}}
+{{- $className = .className -}}
+{{- end -}}
+
+{{if eq .status "success"}}
+	{{svg "octicon-check-circle-fill" $size (printf "ui text green %s" $className)}}
+{{else if eq .status "skipped"}}
+	{{svg "octicon-skip" $size (printf "ui text grey %s" $className)}}
+{{else if eq .status "waiting"}}
+	{{svg "octicon-clock" $size (printf "ui text yellow %s" $className)}}
+{{else if eq .status "blocked"}}
+	{{svg "octicon-blocked" $size (printf "ui text yellow %s" $className)}}
+{{else if eq .status "running"}}
+	{{svg "octicon-meter" $size (printf "ui text yellow job-status-rotate %s" $className)}}
+{{else}}
+	{{svg "octicon-x-circle-fill" $size (printf "ui text red %s" $className)}}
```

Comment:
`ui` class is not needed, see `base.css`.

Reply:
Added a commit https://github.com/go-gitea/gitea/pull/24178/commits/28a20ba40d82b55d506dc0f2ba9fbc288d7744e3 to remove `ui`